director/scripts/test/kinematic_test.py

```diff
@@ -30,21 +34,36 @@ def __init__(self, parent = None):
     def handle_thread(self):
         self.channel = "NETWORK_CASSIE_STATE_DISPATCHER"
```

Comment:
State channel name should be settable(with a default) via command line, or included in the JSON file.